tests/test_ilastik/conftest.py

```diff
@@ -7,3 +7,16 @@
 def data_path():
     conf_path = pathlib.Path(__file__).parent
     return conf_path / "data"
+
+
+def pytest_addoption(parser):
+    parser.addoption("--tiktorch_executable", help="tiktorch executable to use for integration tests")
+
+
+@pytest.fixture
+def tiktorch_executable_path(request):
+    tiktorch_exe_path = request.config.getoption("--tiktorch_executable", None)
```

Comment:
I'd also probably move this to top level conftest (where also `run-legacy-gui` is defined). Or at least have both in the same place.